ConsoleFileManager 2.0/Commands/AddDirectoryCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ConsoleFileManager_2._0.GUI;
+using FileManager.Commands.Base;
+namespace FileManager.Commands;
+
+public class AddDirectoryCommand : FileManagerCommand
+{
+    private readonly IUserInterface _UserInterface;
+    private readonly FileManagerLogic _FileManager;
+    public override string Description => "Для создания вложенной директории введите имя папки.\n│\t\tДля создания папки в другой директории введите путь и имя папки через пробел.";
+    public AddDirectoryCommand(IUserInterface UserInterface, FileManagerLogic FileManager)
+    {
+        _UserInterface = UserInterface;
+        _FileManager = FileManager;
+    }
+    /// <summary>
+    /// Метод для подтверждения
+    /// </summary>
+    /// <param name="dir_path">Путь к файлу</param>
+    private void Aprove(string dir_path, string dir_name)
+    {
+        var toOverride = dir_path.Trim() + dir_name.Trim();
```

Comment:
```suggestion
        var toOverride = Path.Combine(dir_path, dir_name);
```
Безопасный способ "склеить" фрагменты пути. На счёт необходимости `.Trim()` тут - не уверен. Проверьте.